.github/workflows/webhook.yml

```diff
@@ -66,9 +71,10 @@ jobs:
           VERSION=$(make get-version)
           REGISTRY=$(make get-registry)
           IMAGE_NAME=$(make get-image-name)
+          GIT_TAG=$(make get-tag)
           echo "VERSION=$VERSION" >> "$GITHUB_OUTPUT"
           echo "REGISTRY=$REGISTRY" >> "$GITHUB_OUTPUT"
-          echo "TAG_NAME=${{ env.GIT_TAG_PREFIX }}$VERSION" >> "$GITHUB_OUTPUT"
+          echo "TAG_NAME=$GIT_TAG" >> "$GITHUB_OUTPUT"
           echo "FULL_IMAGE_NAME=$REGISTRY/$IMAGE_NAME:$VERSION" >> "$GITHUB_OUTPUT"
```

Comment:
✏️ Now that the image name includes the version, we don't nee this extra `:$VERSION`

Reply:
Good catch, I'll remove it